docs/findings/perf-store-opt-2026-09.md

```
@@ -0,0 +1,116 @@
+# Performance Store Optimizations — 2026-09-06
+
+## Executive summary
+
+#340 (`opt/redb-alloc-reduction`) and #341 (`opt/lmdb-backend`) landed
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)